extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -144,7 +144,7 @@ describes.realWin(
           expect(ampConsent.policyConfig_['_auto_reject']).to.be.ok;
         });
 
-        it('relative checkConsentHref is resolved', function*() {
+        it('relative checkConsentHref is resolved', async function*() {
```

Comment:
you can change `function*` to `() => ` after changing all `yield` to `await`

Reply:
Oops, forgot about generators...